go.mod

```diff
@@ -5,38 +5,38 @@ go 1.13
 require (
 	github.com/MakeNowJust/heredoc v1.0.0 // indirect
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v1.0.0
 	github.com/google/gofuzz v1.1.0
 	github.com/google/uuid v1.1.2
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.14.0
-	github.com/openshift/api v0.0.0-20210412212256-79bd8cfbbd59
-	github.com/openshift/client-go v0.0.0-20210409155308-a8e62c60e930
+	github.com/openshift/api v0.0.0-20211119153416-313e51eab8c8
+	github.com/openshift/client-go v0.0.0-20211025111749-96ca2abfc56c
 	github.com/openshift/cluster-api-provider-gcp v0.0.1-0.20201201000827-1117a4fc438c
-	github.com/openshift/library-go v0.0.0-20210408164723-7a65fdb398e2
+	github.com/openshift/library-go v0.0.0-20210811133500-5e31383de2a7
 	github.com/operator-framework/operator-sdk v0.5.1-0.20190301204940-c2efe6f74e7b
 	github.com/prometheus/client_golang v1.11.0
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
 	github.com/vmware/govmomi v0.22.2
-	golang.org/x/net v0.0.0-20210428140749-89ef3d95e781
+	golang.org/x/net v0.0.0-20210520170846-37e1c6afe023
 	gopkg.in/gcfg.v1 v1.2.3
-	k8s.io/api v0.21.3
-	k8s.io/apimachinery v0.21.3
-	k8s.io/apiserver v0.21.3
-	k8s.io/client-go v0.21.3
-	k8s.io/code-generator v0.21.3
-	k8s.io/klog/v2 v2.8.0
-	k8s.io/kubectl v0.21.0
-	k8s.io/utils v0.0.0-20210722164352-7f3ee0f31471
+	k8s.io/api v0.22.1
+	k8s.io/apimachinery v0.22.1
+	k8s.io/apiserver v0.22.0
+	k8s.io/client-go v0.22.1
+	k8s.io/code-generator v0.22.1
+	k8s.io/klog/v2 v2.10.0
+	k8s.io/kubectl v0.22.0
+	k8s.io/utils v0.0.0-20210819203725-bdf08cb9a70a
```

Comment:
Not sure we should be bumping the K8s dependencies to 1.22 on the 4.8 branch, why did this get changed?

Reply:
I must have replaced at `@master` instead of the intended `@release-4.8`. Fixing now.